test/lib.bash

```diff
@@ -312,21 +311,20 @@ function end_prober {
 }
 
 function upgrade_ocp_cluster {
-  local ocp_target_version upgrade_ocp_image latest_cluster_version
+  local upgrade_ocp_image latest_cluster_version
   upgrade_ocp_image="${1:-}"
 
   if [[ -n "$upgrade_ocp_image" ]]; then
-    ocp_target_version="$upgrade_ocp_image"
     oc adm upgrade --to-image="${UPGRADE_OCP_IMAGE}" \
       --force=true --allow-explicit-upgrade || return $?
+    timeout 7200 "[[ \$(oc get clusterversion version -o jsonpath='{.status.history[?(@.image==\"${upgrade_ocp_image}\")].state}') != Completed ]]" || return $?
```

Comment:
I see. Maybe add additional variable `history_key` to be either `image` or `version`. then it will not be required to copy the entire long command.

Reply:
I'd prefer to keep it as it is. It's not necessary to these micro-optimizations at this level, IMO.

Reply:
Ok